app/views/taxons/_content_list_for_current_taxon.html.erb

```diff
@@ -10,27 +10,55 @@
             <p><%= taxon.description %></p>
           <% end %>
 
+          <ol class="high-volume">
+            <% tagged_content.each_with_index do |content_item, index| %>
+              <% if mainstream_formats.include? content_item.content_store_document_type %>
+                <li>
+                  <h2>
+                    <%= link_to(
+                      content_item.title,
+                      content_item.base_path,
+                      data: {
+                        track_category: is_grid ? 'navGridLeafLinkClicked' : 'navLeafLinkClicked',
```

Comment:
You could delete all these data attributes as we don't track things in the prototype!